crates/orderbook/src/database/orders.rs

```diff
@@ -323,6 +354,51 @@ impl OrderStoring for Postgres {
         order.map(full_order_into_model_order).transpose()
     }
 
+    async fn single_order_with_quote(&self, uid: &OrderUid) -> Result<Option<OrderWithQuote>> {
```

Comment:
Could probably be part of an existing DB test which tests quote fetching in general.